src/fswAlgorithms/attGuidance/sepPoint/_UnitTest/test_sepPoint0.py

```diff
@@ -0,0 +1,226 @@
+#
+#  ISC License
+#
+#  Copyright (c) 2023, Autonomous Vehicle Systems Lab, University of Colorado at Boulder
+#
+#  Permission to use, copy, modify, and/or distribute this software for any
+#  purpose with or without fee is hereby granted, provided that the above
+#  copyright notice and this permission notice appear in all copies.
+#
+#  THE SOFTWARE IS PROVIDED "AS IS" AND THE AUTHOR DISCLAIMS ALL WARRANTIES
+#  WITH REGARD TO THIS SOFTWARE INCLUDING ALL IMPLIED WARRANTIES OF
+#  MERCHANTABILITY AND FITNESS. IN NO EVENT SHALL THE AUTHOR BE LIABLE FOR
+#  ANY SPECIAL, DIRECT, INDIRECT, OR CONSEQUENTIAL DAMAGES OR ANY DAMAGES
+#  WHATSOEVER RESULTING FROM LOSS OF USE, DATA OR PROFITS, WHETHER IN AN
+#  ACTION OF CONTRACT, NEGLIGENCE OR OTHER TORTIOUS ACTION, ARISING OUT OF
+#  OR IN CONNECTION WITH THE USE OR PERFORMANCE OF THIS SOFTWARE.
+#
+
+
+import pytest
+import os
+import inspect
+import numpy as np
+
+filename = inspect.getframeinfo(inspect.currentframe()).filename
+path = os.path.dirname(os.path.abspath(filename))
+bskName = 'Basilisk'
+splitPath = path.split(bskName)
+
+
+# Import all the modules that we are going to be called in this simulation
+from Basilisk.utilities import SimulationBaseClass
+from Basilisk.utilities import RigidBodyKinematics as rbk
+from Basilisk.fswAlgorithms import sepPoint
+from Basilisk.utilities import macros
+from Basilisk.architecture import messaging
+from Basilisk.architecture import bskLogging
+
+
+def computeGamma(alpha, delta):
+
+    if alpha >= 0 and alpha <= np.pi/2:
+        if delta < np.pi/2 - alpha:
+            gamma = np.pi/2 - alpha - delta
+        elif delta > alpha + np.pi/2:
+            gamma = - np.pi/2 - alpha + delta
+        else:
+            gamma = 0
+    else:
+        if delta < alpha - np.pi/2:
+            gamma = - np.pi/2 + alpha - delta 
+        elif delta > 3/2*np.pi - alpha:
+            gamma = alpha + delta - 3/2*np.pi
+        else:
+            gamma = 0
+
+    return gamma
+
+
+# The 'ang' array spans the interval from 0 to pi.  pi is excluded because
+# the code is less accurate around this point;
+ang = np.linspace(0, np.pi, 10, endpoint=False)
+ang = list(ang)
+
+
+@pytest.mark.parametrize("alpha", ang)
+@pytest.mark.parametrize("delta", ang)
+@pytest.mark.parametrize("alignmentPriority", [0])
+@pytest.mark.parametrize("accuracy", [1e-12])
+def test_sepPointTestFunction(show_plots, alpha, delta, alignmentPriority, accuracy):
```

Comment:
You could move these decorators down to the below function and rename it `test_` . This extra function isn't needed now that we've gotten rid of the testMessage/testFailures stuff

Reply:
Done